docs/reference/cluster/reroute.asciidoc

```diff
@@ -129,11 +137,13 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
     Allocate an unassigned replica shard to a node. Accepts `index` and `shard`
     for index name and shard number, and `node` to allocate the shard to. Takes
     <<modules-cluster,allocation deciders>> into account.
---
+======
```

Comment:
I would move this delimiter _after_ the definition for `allocate_empty_primary`. `allocate_stale_primary` and `allocate_empty_primary` are properties for a `commands` object.